romancal/regtest/test_wfi_pipeline.py

```diff
@@ -36,6 +36,7 @@ def test_level2_image_processing_pipeline(rtdata, ignore_asdf_paths):
     output = "r0000101001001001001_01101_0001_WFI01_cal.asdf"
     rtdata.output = output
     args = [
+        "--steps.source_catalog.return_updated_model=True",
```

Comment:
I think we should remove this from here and instead put it in exposure_pipeline.py; i.e., the pipeline should know it needs to call source_catalog with this argument.